flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/nodes/logical/FlinkLogicalTableFunctionScan.java

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
shouldn't we put this into java/** folder
having it in scala is a bit confusing